api/src/components/awaken/awaken.controller.js

```diff
@@ -0,0 +1,3 @@
+const get = (req, res) => res.sendStatus(200);
+
+module.exports = { get };
```

Comment:
I'd probably prefer that we have an index.js file that would store all routes for this folder, then just awaken.js which would contain the functionality. 

up for discussion